dist/ExtUtils-ParseXS/lib/ExtUtils/Typemaps/OutputMap.pm

```diff
@@ -269,16 +269,22 @@ sub targetable {
       m[^
         \s*
         (?:
+            # 1-arg functions
+            sv_set_(?:undef|true|false)
+            \s*
+            \( \s*
+              $target                # arg 1: SV to set
+        |
             # 2-arg functions
-            sv_set([iunp])v
+            sv_set(?:iv|iv_mg|uv|uv_mg|nv|nv_mg|pv|pv_mg|_bool)
             \s*
             \( \s*
               $target                # arg 1: SV to set
               , \s*
               $bal_no_comma          # arg 2: value to use
         |
             # 3-arg functions
-            sv_setpvn
+            sv_set(?:pvn|pvn_mg|pv_bufsize)
```

Comment:
From what I can see this will only match if the *whole* typemap entry is the appropriate call.

For sv_setpv_bufsize() this would result in a returned POK SV where the `length` bytes haven't been set, ie. incomplete code, so I don't see it as reasonable to optimize for.